extensions/amp-bind/0.1/bind-macro.js

```diff
@@ -37,14 +37,14 @@ export class BindMacro {
 
   /**
    * @param {!JsonObject} scope
-   * @param {!Array} args
+   * @param {!Array} arguments
    * @throws {Error} On illegal function invocation.
    * @return {BindExpressionResultDef}
    */
-  evaluate(scope, args) {
-    const copy = /** @type {!JsonObject} */ ({...scope});
+  evaluate(scope, arguments) {
```

Comment:
Naming a variable `arguments` is forbidden in strict mode, which will be a problem for module builds.

Reply:
We should just run strict mode by default. #19380